indexing-service/src/test/java/org/apache/druid/indexing/overlord/hrtr/HttpRemoteTaskRunnerTest.java

```diff
@@ -1606,7 +1607,68 @@ WorkerNodeService.DISCOVERY_SERVICE_KEY, new WorkerNodeService("ip1", 2, "0", Wo
     );
   }
 
-  private HttpRemoteTaskRunner createTaskRunnerForTestTaskAddedOrUpdated(
+  /**
+   * Validate the internal state of tasks withing the task runner
```

Comment:
Thanks for adding the comments! But this seems either a little broad for a test. If you are validating multiple, consider breaking up the test.